src/discussions/discussions-home/DiscussionsHome.test.jsx

```diff
@@ -29,10 +29,16 @@ function renderComponent(location = `/${courseId}/`) {
 
 describe('DiscussionsHome', () => {
   beforeEach(async () => {
+    setConfig({
+      ...getConfig(),
+      FEEDER_PROJECT_ID: 'abc123',
```

Comment:
is that a test ID? `abc123` => `test-id` would be more readable.

Reply:
Yes. It is a test ID. I'll update it.